docs/cloudflare.md

```
@@ -5,31 +5,80 @@ Example using a Cloudflare API Key and corresponding email:
 export CLOUDFLARE_API_KEY=[CLOUDFLARE_API_KEY]
 export CLOUDFLARE_EMAIL=[CLOUDFLARE_EMAIL]
 export CLOUDFLARE_ACCOUNT_ID=[CLOUDFLARE_ACCOUNT_ID]
- ./terraformer import cloudflare --resources=firewall,dns
+./terraformer import cloudflare --resources=dns,firewall,ruleset,access,storage
 ```
 
 or using a Cloudflare API Token:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)